src/lipalightninglib.udl

```diff
@@ -326,6 +326,9 @@ interface Swap {
     [Throws=SwapError]
     SwapAddressInfo create(OpeningFeeParams? lsp_fee_params);
 
+    [Throws=LnError]
+    CalculateLspFeeResponse calculate_swap_lsp_fee_for_amount(u64 amount_sat);
```

Comment:
Given we are within the swap interface, there is no need to have "swap" in the method name :) 

```suggestion
    CalculateLspFeeResponse calculate_lsp_fee_for_amount(u64 amount_sat);
```

Reply:
d7e25c0e7143bb8e9864d23e47f4ea1804d1bd46